TestForSignUp/ElementFiller.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Collections.ObjectModel;
+
+namespace TestForSignUp
```

Comment:
What exactly these tests are doing ? What do we want to test? 
Do we have to test different input? What do we check? Or  we test just that we can submit our registration?

Reply:
We must verify that we can register

Reply:
Yes. But I meant do we need to check every possible data and answer?  Like data registration validation, checks if email is already exist etc.? Or we can just use one test for successful registration?

Reply:
added some checks